packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewFlutterWKWebViewExternalAPI.m

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FWFWebViewFlutterWKWebViewExternalAPI.h"
+#import "FWFInstanceManager.h"
+
+@implementation FWFWebViewFlutterWKWebViewExternalAPI
++ (nullable WKWebView *)webViewForIdentifier:(long)identifier
+                          withPluginRegistry:(id<FlutterPluginRegistry>)registry {
+  FWFInstanceManager *instanceManager =
+      (FWFInstanceManager *)[registry valuePublishedByPlugin:@"FLTWebViewFlutterPlugin"];
+
+  NSObject *instance = [instanceManager instanceForIdentifier:identifier];
```

Comment:
Optional: If you make this `id` instead of `NSObject *` I believe you won't need the cast in the return below.